src/Repository.php

```diff
@@ -553,6 +553,18 @@ public function notActive($name)
     }
 
     /**
+     * Determine if the given module exist and is activated.
+     *
+     * @param string $name
+     *
+     * @return bool
+     */
+    public function isActive($name)
+    {
+        return ($module = $this->find($name)) && $module->active();
```

Comment:
Yes but the actual "active" method throw an error if the module is not present.

The new method let you validate if the module exist and is active.

Example: I need to create a menu based on 3 optional modules. So if a partner install just 2 modules, the menu throw an error with the old active method. The new one just return true or false.